src/Selenium/SeleniumServerAdapter.php

```diff
@@ -21,17 +40,48 @@ public function getLastError()
     }
 
     /**
+     * @return array
+     */
+    public function getServerUrlParts()
+    {
+        return $this->serverUrlParts;
+    }
+
+    /**
+     * @return string
+     */
+    public function getServerUrl()
+    {
+        $parts = $this->serverUrlParts;
+
+        $serverUrl = '';
+        $serverUrl .= isset($parts['scheme']) ? $parts['scheme'] . '://' : '';
```

Comment:
`scheme` and `host` are already validated in `parseServerUrl`. This these checks are unnecessary as they cannot be empty.